scripts/qa_crowbarsetup.sh

```diff
@@ -1211,6 +1219,21 @@ EOF
         /opt/dell/bin/json-edit -a attributes.network.teaming.mode -r -v 5 $netfile
     fi
 
+    # Setup network attributes for JUMBO Frames.
+    if [ -n "${want_mtu_size##+([[:space:]])}" ]; then
+        if [ "$want_mtu_size" == "jumbo" ]; then
+            echo "Value of MTU set in admin and storage nodes is $jumbo_frame_size"
+            /opt/dell/bin/json-edit -a attributes.network.networks.storage.mtu -r -v $jumbo_frame_size $netfile
+            /opt/dell/bin/json-edit -a attributes.network.networks.admin.mtu -r -v $jumbo_frame_size $netfile
+        else
+            echo "Value of MTU set in admin and storage nodes is $want_mtu_size"
+            /opt/dell/bin/json-edit -a attributes.network.networks.storage.mtu -r -v $want_mtu_size $netfile
```

Comment:
this is code duplication, I propose to optimise it to:
(assuming the line above gets dropped)
```
if [[ $want_mtu_size == "jumbo" ]] ; then
    want_mtu_size=8900
fi
if [[ $want_mtu_size ]] ; then
         echo "Setting MTU to custom value of: $want_mtu_size"
         /opt/dell/bin/json-edit -a attributes.network.networks.storage.mtu -r -v $jumbo_frame_size $netfile
         /opt/dell/bin/json-edit -a attributes.network.networks.admin.mtu -r -v $jumbo_frame_size $netfile
fi
```